bag_of_words/code/bag_of_words.py

```diff
@@ -0,0 +1,36 @@
+# Read papers and convert them to bag-of-words representations.
+
+indir = "../data/raw/pubmed-txts-mf/"
+outdir = "../data/processed/"
+from sklearn.feature_extraction.text import CountVectorizer
+vectorizer = CountVectorizer()
+import os
+import numpy as np
+import pickle
+
+# Read papers into a list
+pmids = os.listdir(indir)
+pmids.sort()
+# pmids = pmids[:5]  # For debugging
+corpus = [''] * len(pmids)
+print("Reading corpus ...")
+for i in range(len(pmids)):
+    if i % 10000 == 0:
+        print(str(i) + " docs processed ...")

```

Comment:
I also recommend using newer string formatting methods for Python, e.g. `"%s docs processed".format(i)`